cli/dts/lib.deno.shared_globals.d.ts

```diff
@@ -712,7 +721,18 @@ declare class Worker extends EventTarget {
        * hello world2
        *
        */
-      deno?: boolean;
+      deno?: {
+        namespace?: boolean;
+        permissions?: {
+          env?: "inherit" | boolean;
+          hrtime?: "inherit" | boolean;
+          net?: "inherit" | boolean | string[];
+          plugin?: "inherit" | boolean;
+          read?: "inherit" | boolean | Array<string | URL>;
+          run?: "inherit" | boolean;
+          write?: "inherit" | boolean | Array<string | URL>;
+        };
```

Comment:
When individual fields aren't specified, that's `"inherit"` by default right? And that's the same for the `permissions` field as a whole? So is the `"inherit"` value needed at all?

```suggestion
        permissions?: {
          env?: boolean;
          hrtime?: boolean;
          net?: boolean | string[];
          plugin?: boolean;
          read?: boolean | Array<string | URL>;
          run?: boolean;
          write?: boolean | Array<string | URL>;
        };
```

Reply:
I'd rather have a explicit way of telling the worker it should inherit the permissions. Undefined/ignoring the field might be confusing for some users. @bartlomieju What do you think?

Reply:
Perhaps, but my point was that it's inconsistent to how the `permissions` field is handled. Should that have an `"inherit"` value?

Reply:
You are right, and I think that it should

Reply:
In fact I don't know why it isn't in the d.ts, the API code already expects "inherit" as a valid input